ProcessLib/LiquidFlow/LiquidFlowLocalAssembler.h

```diff
@@ -39,23 +42,30 @@ class LiquidFlowLocalAssemblerInterface
       public NumLib::ExtrapolatableElement
 {
 public:
-    virtual std::vector<double> const& getIntPtDarcyVelocityX(
-        const double /*t*/,
-        GlobalVector const& /*current_solution*/,
-        NumLib::LocalToGlobalIndexMap const& /*dof_table*/,
-        std::vector<double>& /*cache*/) const = 0;
+    LiquidFlowLocalAssemblerInterface(
+        StaggeredCouplingTerm* const coupling_term)
+        : _coupling_term(coupling_term)
+    {
+    }
 
-    virtual std::vector<double> const& getIntPtDarcyVelocityY(
-        const double /*t*/,
-        GlobalVector const& /*current_solution*/,
-        NumLib::LocalToGlobalIndexMap const& /*dof_table*/,
-        std::vector<double>& /*cache*/) const = 0;
+    void setStaggeredCouplingTerm(std::size_t const /*mesh_item_id*/,
+                                  StaggeredCouplingTerm* const coupling_term)
+    {
+        _coupling_term = coupling_term;
+    }
```

Comment:
discussion point: There are no special checks or other procedures of the coupling term, so the `_coupling_term` could be made public saving the "set" method.

Reply:
Just discussed and keep it as it is.